lib/galaxy_test/selenium/test_workflow_run.py

```diff
@@ -583,7 +583,6 @@ def _upload_hello_world_for_input(self, workflow_input, count=1, from_hid=1):
             workflow_input.title(n=i).wait_for_and_clear_and_send_keys(f"hello world.{i + 1}.fastq")
 
         workflow_input.embedded_start_button.wait_for_and_click()
-        workflow_input.status.wait_for_present()
```

Comment:
To actually test this change I would remove `self.history_panel_wait_for_hid_ok(i)`